ee/clickhouse/views/insights.py

```diff
@@ -70,10 +72,17 @@ def funnel(self, request: Request, *args: Any, **kwargs: Any) -> Response:
     @cached_function()
     def calculate_funnel(self, request: Request) -> Dict[str, Any]:
         team = self.team
-        filter = Filter(request=request, data={**request.data, "insight": INSIGHT_FUNNELS})
+        filter = Filter(request=request, data={"insight": INSIGHT_FUNNELS})
 
-        if filter.display == TRENDS_LINEAR:
+        # backwards compatibility - unsure of implications yet
+        # but it's very confusing that funnel trends constant is same as the Trends display constant
+        if not filter.funnel_viz_type and filter.display == TRENDS_LINEAR:
+            filter.funnel_viz_type = FunnelVizType.TRENDS
+
+        if filter.funnel_viz_type == FunnelVizType.TRENDS:
```

Comment:
this checking will probably become more general later on, but fine (i think) as is for now.